src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -24,6 +24,7 @@ import {WriteDataDetailsContext} from 'src/writeData/components/WriteDataDetails
 // Selectors
 import {getOrg} from 'src/organizations/selectors'
 import {getMe} from 'src/me/selectors'
```

Comment:
Note that some of the onboarding experience pages still require a reference to what is, in cloud, the IDPE /me selector for org id and name. Can't remove this at this point, as those states are still part of OSS.